garnbarn_api/models.py

```diff
@@ -45,7 +45,7 @@ class Tag(models.Model):
     color = models.CharField(max_length=10, null=True,
                              blank=True, default=None)
     author = models.ForeignKey(
-        CustomUser, on_delete=models.CASCADE, null=True, default=None)
+        CustomUser, on_delete=models.CASCADE, blank=True, null=True, default=None)
```

Comment:
Should author allow to be None?

Reply:
I think It shouldn't. but actually, you are the one who adds this change in commit fa223f5 . So I'll give this question back to you to answer.

Reply:
For now I think we can leave it like this.